ice_discharge.org

```diff
@@ -161,19 +160,17 @@ After updates, re-run make, and then...
+We have produced an open and reproducible estimate of Greenland ice sheet solid ice discharge from 1986 through last month. Our results show three modes at the total ice-sheet scale: Steady discharge from 1986 through 2000, increasing discharge from 2000 through 2005, steady discharge from 2005 through the recent record. The behavior of individual sectors and glaciers is more complicated. This work was done to provide a 100% reproducible and operational estimate to help constrain mass balance and sea level rise estimates. 
 
 * New in this version
 
-The data were updated approximately every 12 days between the last published version of the paper citep:mankoff_2020_solid and the failure of one of the Sentinel satellites. Updates occur approximately monthly at the moment. Every change is versioned with unique DOIs and posted on the new GEUS data portal at doi:10.22008/promice/data/ice_discharge. The git commit for this version of the paper is \input{|"git describe --always --dirty='*'"}.
+PROMICE ice velocity from Sentinel1 citep:solgaard_2021,solgaard_2021_data is now generated at 200 m rather than 500 m resolution. There is no significant code change associated with this updated data. This results in higher estimated velocities from the smallest glaciers, with a slight increase in overall discharge, and a more than slight increase in the SE sector due to the many small outlet glaciers there.
 
-BedMachine has been updated from v3 to v4. Uncertainty has declined by ~10 % (that is, from ~10 % of total discharge to ~9 %, or from ~50 Gt yr^{-1} to ~45 Gt yr^{-1}). With the BedMachine v4 update, the southeast sector is no longer patched with citet:millan_2018 data.
+The elevation change code and data have been completely rewritten. Previous versions used the GIMP DEM (NSIDC 0645; citet:howat_2014_greenland,NSIDC_0645), a DEM generated from several years of data but with each pixel time-stamped. Elevation change was generated per-pixel using the citet:khan_2016_geodetic elevation change data set. The new workflow uses the PROMICE PRODEM 2019 DEM (CITE winstrup) as the baseline DEM, and annual PRODEMs forward in time from there. When no PRODEM exists (at the moment, for the past year) there is no computed change in elevation. Prior to 2019, we still use citet:khan_2016_geodetic to the beginning of that dataset which is 1995. Prior to 1995 there is no computed change in elevation. Annual DEMs and change in elevations are interpolated linearly and used to estimate a thickness at each velocity timestamp.
```

Comment:
I think there is a mismatch between the stated baseline DEM is from 2019, but in the code 2020 is used?